crates/bevy_core_pipeline/src/bloom/mod.rs

```diff
@@ -512,7 +512,11 @@ impl FromWorld for BloomPipelines {
                                 dst_factor: BlendFactor::One,
                                 operation: BlendOperation::Add,
                             },
-                            alpha: BlendComponent::REPLACE,
+                            alpha: BlendComponent {
+                                src_factor: BlendFactor::One,
+                                dst_factor: BlendFactor::One,
+                                operation: BlendOperation::Max,
+                            },
```

Comment:
For what was this needed?

Reply:
bloom currently outputs alpha of the bloom layer (which is typically small) and destroys the alpha of the input. this made blending behave badly in multi-flush scenarios with bloom.